### PR TITLE
Many updates to CASS site

### PR DESCRIPTION
**Pull request by kelnera**
@osu-cass/cass-web Lots of little updates. Here's what you should see:

- The description for the SmarterBalanced project added

- Lots of little typos fixed

- Lots of fixed links

Please look over the changes and let me know if you see any issues.

### CODE REVIEW

File: content/pages/organization/projects.rst [hunk 1]
```diff
@@ -15,10 +15,10 @@ DevOps
...
 +++++++++
 
-OSL is leveraging OpenStack cloud technology for development and educational
-use. Developers use the OpenStack cluster to easily create virtual machines for
-testing code, while students in the OSL-taught System Administration course
-(CS312) use openstack for class assignments.
+The OSU Open Source Lab (OSL) is leveraging OpenStack cloud technology for
+development and educational use. Developers use the OpenStack cluster to easily
+create virtual machines for testing code, while students in the OSL-taught
+System Administration course (CS312) use openstack for class assignments.
```
Comment (leian7) [inline content/pages/organization/projects.rst]: Does openstack need to be capitalized here too?

Comment (ramereth) [inline content/pages/organization/projects.rst]: Yes. It should be "OpenStack"

**Comment by kelnera** [comment]
@leian7 @ramereth Fixed. Thanks!

**Comment by kelnera** [comment]
@osu-cass/cass-web One last addition: I've included Samar though I don't have his picture yet so it's just his name and bio.

File: content/pages/student-experience/students.rst [hunk 2]
```diff
@@ -56,9 +56,7 @@ users, developers and potential employers. For an idea of some of impact CASS
 student employees have been making, check out our student stories or go to our
 `alumni`_ page.
 
-* Emily Dunham - `Nominated for Women in Open Source Award`_
 * `DevOps Bootcamp`_
 
-.. _Nominated for Women in Open Source Award: http://www.redhat.com/en/about/women-in-open-source
 .. _DevOps Bootcamp: http://devopsbootcamp.osuosl.org/
 .. _alumni: http://cass.oregonstate.edu/alumni/
```
Comment (leian7) [inline content/pages/student-experience/students.rst]: Might want to make the alumni link local instead of outgoing to cass.oregonstate.edu. Otherwise +1

**Pull request merged**